src/mscorlib/src/System/IO/StreamReader.cs

```diff
@@ -529,6 +532,22 @@ private bool IsPreamble()
             return _checkPreamble;
         }
 
+        private StringBuilder AcquireStringBuilder(int capacity)
+        {
+            var sb = this.builder;
```

Comment:
Style comment: Can you please replace the var declaration with the actual type.  We usually do this for readability.